src/_help/bump-cli.md

```diff
@@ -28,164 +35,205 @@ Or, with Yarn via
+    [Breaking] Body attribute modified: energie
 ```
+> You can create as many diffs as you like without being authenticated. This is a **free and unlimited service** provided as long as you use the service fairly.
+{: .info}
 
-You can find your own `my-documentation` slug from your [documentation settings](https://bump.sh/docs).
-
-> You will need to pass your private documentation access token for this command to work. Either with the `--token` flag or via the `BUMP_TOKEN` environment variable. This token can be found from your documentation `settings > CI deployment`page
-{: .warning}
+_Note: You can also test this feature in our dedicated web application at <https://api-diff.io/>._
 
-## Deploy a file
+#### Authenticated diffs related to your Bump.sh documentation
 
-Once your documentation has been updated and merged, you want it to be live for your API users. This is what the `deploy` command is for. When deploying the new version, Bump will analyse your API structure and will generate a changelog item if the API structure has changed.
+From an existing Bump.sh documentation, the `diff` command will retrieve a comparison changelog between your latest published documentation and the given file or URL:
 
 ```shell
-bump deploy path/to/file.json --doc my-documentation
+bump diff path/to/your/file.yml --doc my-documentation --token $DOC_TOKEN
 ```
 
-You can find your own `my-documentation` slug from your [documentation settings](https://bump.sh/docs).
-
-You can also deploy a given file to a different branch of your documentation with the `--branch <branch-name>` parameter. The branch will be created if it doesn't exist. More details about the branching feature is available on [this dedicated help page](/help/branching).
+If you want to compare two unpublished versions of your API document, the `diff` command can retrieve a comparison changelog between two given file or URL, “as simple as `git diff`”:
 
 ```shell
-bump deploy path/to/file.json --doc my-documentation --branch staging
+bump diff path/to/your/file.yml path/to/your/next-file.yml --doc my-documentation --token $DOC_TOKEN
 ```
 
-> You will need to pass your private documentation access token for this command to work. Either with the `--token` flag or via the `BUMP_TOKEN` environment variable. This token can be found from your documentation `settings > CI deployment`page.
-{: .warning}
+Please check `bump diff --help` for full usage details.
 
-## Deploy a folder
+### `bump preview [FILE]`
 
-When using the [Hub](/help/hubs) feature on Bump.sh, you might want to deploy multiple api definition files in a single command. The `deploy` command described in the [previous paragraph](#deploy-a-file) accepts a folder path as argument to do just that:
+
+When writing a documentation, you might want to preview how it renders on Bump.sh. This is precisely the goal of the `preview` command: it will create a temporary documentation with a unique URL, which will be available for a short period.
```

Comment:
```suggestion
When writing documentation, you might want to preview how it renders on Bump.sh. This is precisely the goal of the `preview` command: it will create temporary documentation with a unique URL, which will be available for a short period.
```

Reply:
Do we want to be more explicit about the duration?